### PR TITLE
enable cortex and loki monitoring

### LINKED ISSUE #7223: Cortex & Loki-distributed monitoring in Seed
Monitor Cortex and Loki-distributed backends running in Seed. Provide:
- alerting rules for Cortex and Loki-distributed
- Grafana Dashboards for Cortex and Loki-distributed

### PR DESCRIPTION
**Pull request by sankalp-r**
Signed-off-by: Sankalp Rangare <sankalprangare786@gmail.com>

**What this PR does / why we need it**:

**Which issue(s) this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close the issue(s) when PR gets merged)*:
Fixes #7223 

**Does this PR introduce a user-facing change?**:
```release-note
NONE
```

### CODE REVIEW

**Comment by jiachengxu** [comment]
Looks like you need to rebase with master branch and bump the grafana chart version

**Comment by rastislavs** [comment]
One question to this PR: will these rules be applied automatically for each default KKP installation, even if user cluster MLA is not enabled? If yes, wouldn't that produce false alarms?

**Comment by sankalp-r** [comment]
> One question to this PR: will these rules be applied automatically for each default KKP installation, even if user cluster MLA is not enabled? If yes, wouldn't that produce false alarms?

Yes, you are right.
Same is true for Grafana dashboard for MLA components.
Any suggestions on how we can handle this?

**Comment by rastislavs** [comment]
Maybe we should keep it somehow separately and just document how to install it in the docs?
@wurbanski do you have any idea on this?

File: charts/monitoring/grafana/dashboards/mla/mla-components.json [hunk 1]
```diff
@@ -0,0 +1,2022 @@
+{
+  "annotations": {
+    "list": []
+  },
+  "description": "Mla Deployments/Statefulset Overview",
```
Comment (rastislavs) [inline charts/monitoring/grafana/dashboards/mla/mla-components.json]: Capital `MLA` or even better "User Cluster MLA" - to use the same naming as in the docs: [User Cluster MLA](https://docs.kubermatic.com/kubermatic/v2.18/guides/monitoring_logging_alerting/user_cluster/)

Comment (sankalp-r) [reply charts/monitoring/grafana/dashboards/mla/mla-components.json]: Fixed the description.

**Pull request merged**